mono/tests/coop-array.mak

```diff
@@ -0,0 +1,27 @@
+# This file is not used in automated tests, but is useful for unautomated testing.
```

Comment:
I don't see why we need this file, why is the normal runtime test infrastructure not sufficient?

Reply:
I thought it was useful for AOT but it isn't. Perhaps it isn't better than Makefile.am but I did use it again tonight.

Reply:
Maybe just add a new target to Makefile.am that focuses just on this set of tests?